packages/loader/container-loader/src/utils.ts

```diff
@@ -6,7 +6,13 @@
 import { parse } from "url";
 import { v4 as uuid } from "uuid";
 import { assert, bufferToString, stringToBuffer } from "@fluidframework/common-utils";
-import { ISummaryTree, ISnapshotTree, SummaryType } from "@fluidframework/protocol-definitions";
+import { ISnapshotTree, ISummaryTree, SummaryType } from "@fluidframework/protocol-definitions";
+
+// This will got in common place to loader and runtime as it used by both.
+export interface ISnapshotTreeWithBlobContents extends ISnapshotTree {
```

Comment:
This interface formalizes contract and reduces chance of extra conversions (based64 and back) which is good thing.
But does usage of this interface passes through compat boundary, or runtime and loader have independent usage? I'd guess that convertSnapshotTreeToSummaryTree() is the primary consumer if it's the later, but isn't usage of ContainerStorageAdapter() ensures that runtime does not need to know about it (at least in first approximation, there is a need to revisit sync -> async changes when loading local data stores).